### PR TITLE
Requesting module semantic versioning

### PR DESCRIPTION
**Pull request by BrandonRoehl**
https://golang.org/cmd/go/#hdr-Module_compatibility_and_semantic_versioning

Requesting of changing the import path to `/v4` since version numbers greater than `v1.X.X` are no longer supported with the `+incompatible` flag as of Go 1.13 https://golang.org/doc/go1.13 section **Version validation** https://blog.golang.org/v2-go-modules

And an example without a sub directory so long as they are tagged https://github.com/google/go-github

We have to currently modify the version to use Pseudo-versions https://golang.org/ref/mod

Example:
- To example in order to include v4.9.7 we would use `github.com/Venafi/vcert v0.0.0-20200514165030-4f70baad986f`
- Example of a path change v4.9.8 could be included as `github.com/Venafi/vcert/v4 v4.9.8`

> **Note:** meant to be a discussion and feedback is desired.
> This is proposed since `github.com/Venafi/vcert v4.9.7+incompatible` no longer works in go 1.13

### CODE REVIEW

**Comment by tr1ck3r** [comment]
@asquith-venafi @venafibonjo this seems related to https://github.com/Venafi/vcert/issues/67

File: README.md [hunk 1]
```diff
@@ -32,13 +32,13 @@ Custom Fields and Instance Tracking require TPP 18.2 or higher, and Token Authen
...
 ```sh
-go get github.com/Venafi/vcert
+go get github.com/Venafi/vcert/v4
 ```
 
 or
 
 ```sh
-git clone https://github.com/Venafi/vcert.git $GOPATH/src/github.com/Venafi/vcert
+git clone https://github.com/Venafi/vcert/v4.git $GOPATH/src/github.com/Venafi/vcert/v4
```
Comment (lhdomenech) [inline README.md:41]: the git url is invalid, the repo is still https://github.com/Venafi/vcert.git

File: README.md [hunk 2]
```diff
@@ -9,8 +9,8 @@ We welcome and appreciate all contributions._
 
 # VCert
 
-[![GoDoc](https://godoc.org/github.com/Venafi/vcert?status.svg)](https://godoc.org/github.com/Venafi/vcert)  [![Go Report Card](https://goreportcard.com/badge/github.com/Venafi/vcert)](https://goreportcard.com/report/github.com/Venafi/vcert)
-[![Used By](https://sourcegraph.com/github.com/Venafi/vcert/-/badge.svg)](https://sourcegraph.com/github.com/Venafi/vcert?badge)
+[![GoDoc](https://godoc.org/github.com/Venafi/vcert/v4?status.svg)](https://godoc.org/github.com/Venafi/vcert/v4)  [![Go Report Card](https://goreportcard.com/badge/github.com/Venafi/vcert/v4)](https://goreportcard.com/report/github.com/Venafi/vcert/v4)
+[![Used By](https://sourcegraph.com/github.com/Venafi/vcert/v4/-/badge.svg)](https://sourcegraph.com/github.com/Venafi/vcert/v4?badge)
```
Comment (lhdomenech) [inline README.md:13]: I believe the sourcegraph url is based on the repo url so v4 should not be part of that

**Review by lhdomenech — CHANGES_REQUESTED** [review]
Looks good. Just a few minor things in readme.

File: README.md (hunk 1 above)
Comment (BrandonRoehl) [reply README.md]: These have been updated and added the note from https://blog.golang.org/using-go-modules about being outside of `$GOPATH/src` when developing in module mode. Wondering if it worth adding the note for `replace github.com/Venafi/vcert => ../path` note for developing go mod locally in other repos when working with modules in go 1.13

Comment (lhdomenech) [reply README.md]: I don't think it is needed, this is standard practices by now.

**Review by lhdomenech — APPROVED** [review]

**Pull request merged**